contracts/smart-account/modules/AccountRecoveryModule.sol

```diff
@@ -236,23 +157,28 @@ contract AccountRecoveryModule is
         assembly {
             innerSelector := mload(add(innerCallData, 0x20))
         }
+
         bool isValidAddingRequestUserOp = (innerSelector ==
             this.submitRecoveryRequest.selector) &&
             (dest == address(this)) &&
             callValue == 0;
-        if (
-            isValidAddingRequestUserOp != //this a userOp to submit Recovery Request
-            (_smartAccountSettings[msg.sender].securityDelay == 0) //securityDelay is 0,
-        ) {
+        if (isValidAddingRequestUserOp)
```

Comment:
can rename to something more readable like isUserOpValidForRecoveryRequestSubmission